File: Facebook.NET/GraphAPI.Education.cs
using System;
using System.Runtime.Serialization;

namespace Facebook
{
    public static partial class GraphAPI
    {
        [DataContract]
        public sealed class Education
        {
            [DataMember(Name = "school")]
            public Domain School { get; set; }

            [DataMember(Name = "year")]
            public Domain Year { get; set; }

            [DataMember(Name = "type")]
            public string Type { get; set; }

            [DataMember(Name = "concentration")]
            public GraphAPI.Domain Concentration { get; set; }
        }
    }
}

File: Facebook.NET - VB/Contracts/GraphAPI.Education.vb
Imports System
Imports System.Runtime.Serialization

Namespace Facebook
    Public NotInheritable Partial Class GraphApi
        <DataContract()>
        Public NotInheritable Class Education
            <DataMember(Name:="school")>
            Public Property School() As Domain
            <DataMember(Name:="year")>
            Public Property Year() As Domain
            <DataMember(Name:="type")>
            Public Property Type() As String
            <DataMember(Name:="concentration")>
            Public Property Concentration() As GraphApi.Domain
        End Class
    End Class
End Namespace